kncaputo/urban-stargazer
# Fix/local storage

## What's this PR do?
- Fixes save image feature and adds `remove from saved` to `Discover`
- Enables users to toggle between save and remove photo - closes #9 
- Exports `saveToLocalStorage` as a helper function into `utilities`
- Adds `filterData` to `utilities`
- Adds basic styling to saved image cards
- Deletes cards from `Saved` on click (removes from local storage) (might want to change this at a later iteration to have click to view and have the delete option available from within the view)
- Fixes users ability to save a photo twice - closes #8 
- closes #3 

## Where should the reviewer start?
- Start by reviewing `utilities` and `Discover`

## How should this be manually tested?
- Test in the browser by going to the `Discover` component and toggling the `Save Photo` and `Remove from Saved` button.
- Check that photos appear in `Save` component
- In `Save`, click photo to delete
- Check local storage that photo is being deleted

## Screenshots/GIFs
<img width="946" alt="Screen Shot 2021-01-16 at 4 14 31 PM" src="https://user-images.githubusercontent.com/68157759/104823151-f3a27580-5815-11eb-84b0-8b0a733e93af.png">
<img width="945" alt="Screen Shot 2021-01-16 at 4 14 21 PM" src="https://user-images.githubusercontent.com/68157759/104823153-f4d3a280-5815-11eb-8dd8-1c71908e79fa.png">
